server/index.ts

```diff
@@ -23,25 +26,138 @@ const server = new ApolloServer({
     resolvers,
 });
 
+const clientId = process.env.GITHUB_CLIENT_ID;
+const clientSecret = process.env.GITHUB_CLIENT_SECRET;
+
+app.get('/auth', (req: Request, res: Response) => {
```

Comment:
It's preferable for our server index.ts file to be cleaner, so before merging can you migrate the auth functionality to an Auth router and controller?

Reply:
Yup!